config/fam_memoryserver_config.yaml

```diff
@@ -0,0 +1,4 @@
+# MemoryServer Config file
```

Comment:
Need to specify memory server IP address and grpc port for memory service startup.
Also need to specify libfabric port and provider options.